docs/products/clickhouse/howto/configure-tiered-storage.rst

```diff
@@ -0,0 +1,92 @@
+Configure data retention thresholds in Aiven for ClickHouse®'s tiered storage |beta|
+====================================================================================
+
+Learn to control how your data is distributed between storage devices in the tiered storage of an Aiven for ClickHosue service. Check out how to configure tables so that your data is automatically written either to SSD or the object storage as needed.
+
+About data retention control
+----------------------------
+
+If you have the tiered storage feature :doc:`enabled </docs/products/clickhouse/howto/enable-tiered-storage>` on your Aiven for ClickHouse service, your data is distributed between two storage devices (tiers). The data is stored either on SSD or in the object storage, depending on whether and how you configure this behavior. If you don't, by default the data is moved from SSD to the object storage when SSD reaches 80% of its capacity (default size-based data retention policy).
+
+You may want to change this default data distribution behavior and configure your table's design so that all the data that is there on your SSD for over a specific period of time is moved to the object storage, regardless of how much of the SSD capacity is still available.
+
+To enable this time-based data distribution mechanism, you can set up a retention policy (threshold) on a table level by using the TTL (time-to-live) clause. You can include the TTL clause after a column definition and/ or at the end of the table definition. For data retention control purposes, the TTL clause uses the following:
```

Comment:
The TTL clause over columns and tables serve different purposes: after the column TTL has expired the row is deleted, whereas after the table TTL has expired the row is moved depending on the storage policy.
IIUC we're interested in table-level TTL for this article. It would be clearer if we removed the column TTL reference and examples.